contributing.rst

```diff
@@ -145,7 +145,15 @@ deploy the built charms to a cloud environment of your choosing. There are two
 primary ways to deploy the charms after building them:
 
 - use Juju to deploy the built charms directly e.g.
-  ``juju deploy ./slurmd.charm``
+  ``juju deploy ./slurmd.charm``.
+
+  .. note::
+
+     ``slurmctld`` requires an ``etcd`` resource to be deployed. Download it
```

Comment:
Should the charm download the etcd if it isn’t supplied as a charm resource?

Reply:
The reason to use a resource is that the charm does not download stuff from the internet, but from the controller. Sometimes the charm might not have access to the public internet (corporate firewall). Do we want to handle all those scenarios in the charm code?